tutor_webhook_receiver/plugin.py

```diff
@@ -2,14 +2,45 @@
 import os
 import pkg_resources
 
+from .__about__ import __version__
 
 templates = pkg_resources.resource_filename(
     "tutor_webhook_receiver", "templates"
 )
 
-config = {}
+config = {
+    "add": {
+        "DB_PASSWORD": "{{ 8|random_string }}",
+        "DJANGO_SECRET_KEY": "{{ 20|random_string }}",
+        "EDX_OAUTH2_SECRET": "{{ 8|random_string }}",
+    },
+    "defaults": {
+        "VERSION": __version__,
```

Comment:
I know the cookiecutter sets this, but I wonder what purpose it serves for this to be a *default?* I mean, what exactly is the purpose of being able to *set* this variable?

Reply:
Good question :thinking: I doubt it is used anywhere. I remove it.